tests/gem5/gpu/test_gpu_pannotia.py

```diff
@@ -0,0 +1,344 @@
+    "ecology1.graph": "https://storage.googleapis.com/dist.gem5.org/dist/develop/datasets/pannotia/color/ecology1.graph",
+    "256_16384.gr": "https://storage.googleapis.com/dist.gem5.org/dist/develop/datasets/pannotia/floydwarshall/256_16384.gr",
+    "512_65536.gr": "https://storage.googleapis.com/dist.gem5.org/dist/develop/datasets/pannotia/floydwarshall/512_65536.gr",
+    "G3_circuit.graph": "https://storage.googleapis.com/dist.gem5.org/dist/develop/datasets/pannotia/mis/G3_circuit.graph",
+    "ecology1.graph": "https://storage.googleapis.com/dist.gem5.org/dist/develop/datasets/pannotia/mis/ecology1.graph",
+    "coAuthorsDBLP.graph": "https://storage.googleapis.com/dist.gem5.org/dist/develop/datasets/pannotia/pagerank/coAuthorsDBLP.graph",
+    "USA-road-d.NY.gr.gz": "http://www.diag.uniroma1.it/challenge9/data/USA-road-d/USA-road-d.NY.gr.gz",
+}
+
+
+if not os.path.isdir(resource_path):
+    os.makedirs(binary_path)
+    os.makedirs(dataset_path)
+
+    for name in binary_links.keys():
+        urlretrieve(binary_links[name], f"{binary_path}/{name}")
+    for name in dataset_links.keys():
+        urlretrieve(dataset_links[name], f"{dataset_path}/{name}")
+
+    with gzip.open(f"{dataset_path}/USA-road-d.NY.gr.gz", "rb") as f_in:
+        with open(f"{dataset_path}/USA-road-d.NY.gr", "wb") as f_out:
+            shutil.copyfileobj(f_in, f_out)
+    os.remove(f"{dataset_path}/USA-road-d.NY.gr.gz")
+
+if len(os.listdir(binary_path)) < len(binary_links):
+    testlib.log.test_log.warn(
+        "One or more binaries for the Pannotia GPU tests are missing! Try deleting gpu-pannotia-resources and rerunning."
+    )
+if len(os.listdir(dataset_path)) < len(dataset_links):
+    testlib.log.test_log.warn(
+        "One or more dataset files for the Pannotia GPU tests are missing! Try deleting gpu-pannotia-resources and rerunning."
+    )
+
+
+gem5_verify_config(
+    name="gpu-apu-se-pannotia-bc-1k-128k",
+    fixtures=(),
+    verifiers=(),
+    config=joinpath(config.base_dir, "configs", "example", "apu_se.py"),
+    config_args=[
+        "-n3",
+        "--mem-size=8GB",
+        "-c",
+        joinpath(binary_path, "bc.gem5"),
+        "--options",
+        joinpath(dataset_path, "1k_128k.gr"),
+    ],
+    valid_isas=(constants.vega_x86_tag,),
+    valid_hosts=(constants.host_gcn_gpu_tag,),
+    length=constants.very_long_tag,
+)
+
+gem5_verify_config(
+    name="gpu-apu-se-pannotia-bc-2k-1M",
+    fixtures=(),
+    verifiers=(),
+    config=joinpath(config.base_dir, "configs", "example", "apu_se.py"),
+    config_args=[
+        "-n3",
+        "--mem-size=8GB",
+        "-c",
+        joinpath(binary_path, "bc.gem5"),
+        "--options",
+        joinpath(dataset_path, "2k_1M.gr"),
+    ],
+    valid_isas=(constants.vega_x86_tag,),
+    valid_hosts=(constants.host_gcn_gpu_tag,),
+    length=constants.very_long_tag,
+)
+
+gem5_verify_config(
```

Comment:
Previously for color we were running smaller inputs (https://github.com/gem5/gem5/blob/stable/tests/deprecated/weekly.sh#L275) to make the tests complete in a more tractable time.  I believe these test sizes you have will likely run for multiple days.  If that is ok, then fine but just an FYI.